topi/python/topi/vision/nms.py

```diff
@@ -21,6 +21,55 @@
 from tvm import hybrid
 from ..sort import argsort
 
+
+@hybrid.script
+def hybrid_rearrange_idx(data):
```

Comment:
I think we can merge this with hybrid_rearrange_out by adding a branch to deal with 2-D case?